### PR TITLE
Eliminate duplicate certificates in X509CAChain

### PR DESCRIPTION
**Pull request by hiyosi**
Signed-off-by: Tomoya Usami <tousami@zlab.co.jp>

**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**

UpstreamAuthority vault plugin

**Description of change**

Since Vault v1.11.0, the signed ca certificate appears within the ca_chain field of the sign-intermediate API response.
The vault plugin eliminates duplicate certificates from X509CaChain response.

see: https://github.com/hashicorp/vault/blob/v1.11.0/changelog/15524.txt

**Which issue this PR fixes**

### CODE REVIEW

File: pkg/server/plugin/upstreamauthority/vault/vault_client.go [hunk 1]
```diff
@@ -101,11 +101,11 @@ type Client struct {
 // SignCSRResponse includes certificates which are generates by Vault
 type SignCSRResponse struct {
 	// A certificate requested to sign
-	CertPEM string
-	// A certificate of CA(Vault)
 	CACertPEM string
+	// A certificate of CA(Vault)
+	UpstreamCACertPEM string
```
Comment (hiyosi) [inline pkg/server/plugin/upstreamauthority/vault/vault_client.go:106]: The field names has been changed in this commit.
But if it is better or prefer to keep the original, please comment.

**Review by amartinezfayo — APPROVED** [review]
Thank you, @hiyosi!

**Comment by azdagron** [comment]
Will existing deployments that have not moved to Vault 1.11.x yet still function with this change?

**Comment by hiyosi** [comment]
@azdagron 

Yes, it is compatible.
From 1.11.x, they just added the signed ca certificate within ca_chain field.

## Sample response from Vault sign-intermediate endpoint.

### Vault v1.10.5                         

```
{
    "request_id": "202071f6-37a2-3ad2-57f0-801ac670f162",
    "lease_id": "",
    "lease_duration": 0,
    "renewable": false,
    "data": {
        "ca_chain": [
            "-----BEGIN CERTIFICATE-----\nMIIDNDCCAhygAwIBAgIUaBQLlC/NxTP72aJTMfiUuN8zdrEwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MTk0NzE0WhcNMjcw\nODI1MTk0NzQ0WjAtMSswKQYDVQQDEyJteXZhdWx0LmNvbSBJbnRlcm1lZGlhdGUg\nQXV0aG9yaXR5MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAw8sGDeao\niq9gL3Mp1Vxa/2Ybank4bEThu6eB36/XtP+ycQ1sePKyDhAqVvwI9Mr7GQ0a8Q/l\nvk6sMW2oMHlXEkeX7pdZevIKAeZYLCBdoWlOwhRrJhRb/4nX2prWQWwvU93Xm64V\nacqt47U07FuFhALokT5iXsCnICbVrpDh03vY508KpbV+N+8+86c8jDrpFburwZv0\n3gWh4uurDg8gVybHzvIhD+YiSIBfm04RgGoJ+SMBsr898DNsO3uppw3PKDR044HY\nbHiUkIpwMrBzceTZXPqOtNhkEUF4KBIwqHKBxrwJzz56enXCWzfYGVJueX7pLnRG\nnN8N9S7k7Y7DywIDAQABo2MwYTAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUw\nAwEB/zAdBgNVHQ4EFgQUtzQi0sRD/W/NjJH87L/mg7R1JMcwHwYDVR0jBBgwFoAU\nQijI1eQoVHuhpC08hHbRpJEG4/MwDQYJKoZIhvcNAQELBQADggEBACDzp+YA6MW/\nb10UKU540Bw+XAJuGor2x5lA25mnmpU4C4fB3g9SQzMHLeS5oHY2lfR1ys9jDfNz\nNUHTPvQlu1y2fP5iEFw0gPU20N3fRwCTQqeVufSlroMQoMtSgQ61WMYIEgOjiKa2\nk7lZJn+EHoXCc6YX81FO1ECdS+jGAmYzetH31t4dpBc25mbPJOyslFZYCCUwMne5\nlUOEIps8myqOanOFLrq2T3rjKjW7vk0eIIupI7Xb9oSpdzz8XNkoUic9Gho4anhG\n9ETJFozjcZXHIRA38h5Xcp/SQXrsW6mKOXXP+fgt1t80SckP6pKrLgOqSCa66CrX\nfE8Qw0UMDPs=\n-----END CERTIFICATE-----",
            "-----BEGIN CERTIFICATE-----\nMIIDNTCCAh2gAwIBAgIUCva9JG18Jvnw3OUAYpfdn7TQfx8wDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MTk0NTI2WhcNMzIw\nODIzMTk0NTU2WjAWMRQwEgYDVQQDEwtteXZhdWx0LmNvbTCCASIwDQYJKoZIhvcN\nAQEBBQADggEPADCCAQoCggEBAKeQPo95jRrguMoQgK71OR1H4leBylP/dRcapbQx\n/Ku517jG9KGn7MOYRW2cFnGAYIBjeHVdEeM4tpArmW9AtbHppQCuBkAy3YftMfUw\nuVVeOLJHHIwZeDNuixmyGoGkyNiZE3cIufeXJ2x+QSTC7oAmMTonYxzItRywywDE\nDv/puYeJlOuOME3N6A/bdU6NTvkg3E3O34g91dHnZxglKxrOOdYoNorAH2AnhXGR\n6twohxiFN3rTXRTQOSU/OBag6ToamEpaXXTJnfTPPvDuaXu8/OpaE+BnRZW76q2I\n+s0NszQgP3lwY0XorfBtc5YwkevqnfzA3iPpnGiTNGSSMYECAwEAAaN7MHkwDgYD\nVR0PAQH/BAQDAgEGMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFEIoyNXkKFR7\noaQtPIR20aSRBuPzMB8GA1UdIwQYMBaAFEIoyNXkKFR7oaQtPIR20aSRBuPzMBYG\nA1UdEQQPMA2CC215dmF1bHQuY29tMA0GCSqGSIb3DQEBCwUAA4IBAQAg/AYkCdv2\nyTIK7RfAlE83S3cWkA3dk8zAtHwp6m8IbPVmsGpbg283v8UmW2G4ByAuVn413h7z\n1wPiUar4gKXRkpDCTzRSWTgs08mISp99otv+4ZcYHCL8y600ZyGb8EYcMbqfRqRo\ngB/kujs0mG9p8mf4PoTElwrKBTwd3GvU7P3/WJeaQdWFzMdWJeXioeeim7NjX988\nioXJi5lSrFWQWzV7H9jL1ynSWn5gNvhDtjWFAuj6gVDVkKzTQmKdC+eH7sbRRpKH\ndo/yhHz+ovTscozHduHokxf8aqhwddUrvlDZCa2qhxAK1A1gWQ2B0Ic/mhO5PLLD\n4TFKLiPUOIH7\n-----END CERTIFICATE-----"
        ],
        "certificate": "-----BEGIN CERTIFICATE-----\nMIICUzCCATugAwIBAgIUHZAQ0qsY0Epey5Nwub5NTt42rJkwDQYJKoZIhvcNAQEL\nBQAwLTErMCkGA1UEAxMibXl2YXVsdC5jb20gSW50ZXJtZWRpYXRlIEF1dGhvcml0\neTAeFw0yMjA4MjYyMDAyMzlaFw0yMzAxMjMyMDAzMDlaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAATLBt8xHYkdR4kuDnNM4/vLex+vyVa8rzB3cMv7NE9N57lh\n2OHcABZbObftau9bh25VaX599fwk/EwUumGE2wnjo2MwYTAOBgNVHQ8BAf8EBAMC\nAQYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUeS8mVKwuJN2vr6ctq9rW/qhm\njcowHwYDVR0jBBgwFoAUtzQi0sRD/W/NjJH87L/mg7R1JMcwDQYJKoZIhvcNAQEL\nBQADggEBAEBNme6wEXGB5yqrZAYBzj+eLnR/gwxRrNQj+fXqKAyXnM3RpDIvTfPc\n4W7DM19TlE5hGV8FdexU3KFZPGr50eKj1giecQ4dyWFJUVQsvSNv0OWaqXl22Zgc\nDaPHPQM8RwADhy3jmSWlopQD7DgrgNoS9OcGNYEeh8xQDKyig9KFe/rYOulvNUB/\ne7VKkQKXmyFoaedY5UI150VSdyVPL6ZshMb4yB9r1PIm21pENHTTHdhvTRdUxqhD\nv2km5hoz0eXaYc1MNqO34hm+e64vfU7EiTEAgTpx1QWLhYgDnE6gO4Wy0wVxVV0x\nklfgLHW5+lbo6Fcl7XXCH1qEP5wFaZ8=\n-----END CERTIFICATE-----",
        "expiration": 1674504189,
        "issuing_ca": "-----BEGIN CERTIFICATE-----\nMIIDNDCCAhygAwIBAgIUaBQLlC/NxTP72aJTMfiUuN8zdrEwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MTk0NzE0WhcNMjcw\nODI1MTk0NzQ0WjAtMSswKQYDVQQDEyJteXZhdWx0LmNvbSBJbnRlcm1lZGlhdGUg\nQXV0aG9yaXR5MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAw8sGDeao\niq9gL3Mp1Vxa/2Ybank4bEThu6eB36/XtP+ycQ1sePKyDhAqVvwI9Mr7GQ0a8Q/l\nvk6sMW2oMHlXEkeX7pdZevIKAeZYLCBdoWlOwhRrJhRb/4nX2prWQWwvU93Xm64V\nacqt47U07FuFhALokT5iXsCnICbVrpDh03vY508KpbV+N+8+86c8jDrpFburwZv0\n3gWh4uurDg8gVybHzvIhD+YiSIBfm04RgGoJ+SMBsr898DNsO3uppw3PKDR044HY\nbHiUkIpwMrBzceTZXPqOtNhkEUF4KBIwqHKBxrwJzz56enXCWzfYGVJueX7pLnRG\nnN8N9S7k7Y7DywIDAQABo2MwYTAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUw\nAwEB/zAdBgNVHQ4EFgQUtzQi0sRD/W/NjJH87L/mg7R1JMcwHwYDVR0jBBgwFoAU\nQijI1eQoVHuhpC08hHbRpJEG4/MwDQYJKoZIhvcNAQELBQADggEBACDzp+YA6MW/\nb10UKU540Bw+XAJuGor2x5lA25mnmpU4C4fB3g9SQzMHLeS5oHY2lfR1ys9jDfNz\nNUHTPvQlu1y2fP5iEFw0gPU20N3fRwCTQqeVufSlroMQoMtSgQ61WMYIEgOjiKa2\nk7lZJn+EHoXCc6YX81FO1ECdS+jGAmYzetH31t4dpBc25mbPJOyslFZYCCUwMne5\nlUOEIps8myqOanOFLrq2T3rjKjW7vk0eIIupI7Xb9oSpdzz8XNkoUic9Gho4anhG\n9ETJFozjcZXHIRA38h5Xcp/SQXrsW6mKOXXP+fgt1t80SckP6pKrLgOqSCa66CrX\nfE8Qw0UMDPs=\n-----END CERTIFICATE-----",
        "serial_number": "1d:90:10:d2:ab:18:d0:4a:5e:cb:93:70:b9:be:4d:4e:de:36:ac:99"
    },
    "warnings": null
}
```

## Sample response from Vault

### Vault v1.11.2
```
{
    "request_id": "96113f8c-2e3c-f36a-8948-608e632890d4",
    "lease_id": "",
    "lease_duration": 0,
    "renewable": false,
    "data": {
        "ca_chain": [
            "-----BEGIN CERTIFICATE-----\nMIICUzCCATugAwIBAgIUXOMH5Mw5jQGxFjrd2oXeKgWfkhkwDQYJKoZIhvcNAQEL\nBQAwLTErMCkGA1UEAxMibXl2YXVsdC5jb20gSW50ZXJtZWRpYXRlIEF1dGhvcml0\neTAeFw0yMjA4MjYyMDA4MThaFw0yMzAxMjMyMDA4NDhaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAATLBt8xHYkdR4kuDnNM4/vLex+vyVa8rzB3cMv7NE9N57lh\n2OHcABZbObftau9bh25VaX599fwk/EwUumGE2wnjo2MwYTAOBgNVHQ8BAf8EBAMC\nAQYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUeS8mVKwuJN2vr6ctq9rW/qhm\njcowHwYDVR0jBBgwFoAU0TbPqPx2m/VuI1KB8cLWBP64GKUwDQYJKoZIhvcNAQEL\nBQADggEBAJxJz7sDIa2SMjIsRVPfiGKf9XJQCM23MiGnz7pFVHlm5HOOkZuBpMmQ\n7oUj/1BcdBgCOAd38nzEOrpmNXBkbNTOgiqhIInPGdo4whxhSYbYi7WxJMiYFztE\n6AvbDa6Xj5pNux2fFV2Ur5iudVVCB/p3nyMHvlYqL43xyDG5BJJpDty/af9fcxUR\ny21C1z9DX2lzMMopdcPJ/W5H4jwKi/M6WtwlY5rLQ5+4HS8db5W/gcXv1uFf/OEt\nf4RtpgibBknYBsBs8jRn+7/NJw3hYwPofrWUbdk2r1MBUPYNUraxqMuaWNkDhg4W\n3Vf07lgG1Rf1v7pEItvEv0TNylu+fBs=\n-----END CERTIFICATE-----",
            "-----BEGIN CERTIFICATE-----\nMIIDNDCCAhygAwIBAgIUG4iQ2d5V83due1fJTqnEx06D3oYwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MjAwNTAxWhcNMjcw\nODI1MjAwNTMxWjAtMSswKQYDVQQDEyJteXZhdWx0LmNvbSBJbnRlcm1lZGlhdGUg\nQXV0aG9yaXR5MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAq3Dw7lTU\nEZqhOPOoOZtfevxBz01PFx6BYsi3JTA7mpeSDIre0cKanwjiYc3u9z0nQF5W1Jr5\natkDUcMTOWCALAxRg5vWOq/2bDuv2lLk9f9FYIgdDQHMFQwqq4D0oGX+/ZFBVBAH\nHIYn8nMgnTnBCKzNFqHSQxCuSa42MlC1g6OO7z87sfL1wn5Inhn7OP6eoTzv5dvv\nefH6Mhvv2KvulxdA/imTaPr3wuEu1xmgg0sczUxZxIwQGU/uX+tL1Brwfow3uWGN\ngATknWly71Hp4RdeM+fFg7Or2otn6kimFV10A6kWAao1+CI7evAaPaqVXtrNGSRe\nWhZtVSAiGiZ0/QIDAQABo2MwYTAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUw\nAwEB/zAdBgNVHQ4EFgQU0TbPqPx2m/VuI1KB8cLWBP64GKUwHwYDVR0jBBgwFoAU\nUO7+pScrxAIZHhlgkoToBJHxOUEwDQYJKoZIhvcNAQELBQADggEBAEAVoshZnNsG\nebCxhzqmeAEWOV/i5SxkU4XkYumAHNlpzA6HLB1ieegRDPiMqxfiQkCCUs7WVj7f\nxnI1pZi0ZqzpR54ROJn+WqTPVp4xOxdheWjzfLzkOIfSmZtYDvXqFYS7Wghu3c/K\nno1/fIIiesAN43hzDMT9HhdLQSkCl4Lh5x8SwSZYZ1ATK/iUOwVIIBhvr9nLm+XF\ncFL89bIw9Y2g9VMi5kwrEJoiSIaq6eQkw048C/bH/sV5IhdG5zOY3J7eDxz+dY87\n/7Er+WUprmgcrp2pnwwCy9zJ3k9rzMGao9TPaCTUNl685uGqqNHZCOPscdX/v6Tz\nd5cMOisXvU8=\n-----END CERTIFICATE-----",
            "-----BEGIN CERTIFICATE-----\nMIIDNTCCAh2gAwIBAgIUVZZ6PJMce9tuC/suFzHppEc6BFIwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MjAwNDA4WhcNMzIw\nODIzMjAwNDM4WjAWMRQwEgYDVQQDEwtteXZhdWx0LmNvbTCCASIwDQYJKoZIhvcN\nAQEBBQADggEPADCCAQoCggEBAMi18Nl5EJleJswiEc+zyFoEsmiQajRBl+4/Y5eu\nv9MUvoc1Lh93ace+Uk8Nb7ZY/zc/gWaO0NYc8THUkhaCP6yMgscgBAUK5rlzUZaL\nQsfrixZISZNziVAkMLxu82ueHSrFpC0NNDT00NLq7sNCB7KKsmsCGN2GwWou2T0y\n2oOuZg/m9/Lq2lj9YYgjNgVDL27RwBeYYqsrw3MJC4aSmsqnLGueH8aP/ZtJKKR5\nHmv94nsVDEysWE2Ju8IFPvDwxwvpl9DCHWLCqBcnJvvBqK/DAAZY3oitcq1WM5A6\nhAWCPL/83K3euNOv3MXlSw6fjtU7cKgaUQquyyYG67DUcjkCAwEAAaN7MHkwDgYD\nVR0PAQH/BAQDAgEGMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFFDu/qUnK8QC\nGR4ZYJKE6ASR8TlBMB8GA1UdIwQYMBaAFFDu/qUnK8QCGR4ZYJKE6ASR8TlBMBYG\nA1UdEQQPMA2CC215dmF1bHQuY29tMA0GCSqGSIb3DQEBCwUAA4IBAQCqZjEW6d6+\n4RcfM0XFjZVnmITyxGUOi+2g0rhaEb/UiI+KX5Z1HMjnrmrLbRi4vxXtBNRQiLyw\nwepiSzKIvozwJx9nyBO4W5srHWx7m1pJKcH273041h2GWHct9Y4VUNBOCCdqC/JV\n10V/Y8wkvKpiYAoHXDyNz/F460Yk06nrBr9bBnrybjWiFa80Fiqt4CKwphGeCyfG\npMn/6zjHEOnRMS4Em6fYSjSJkqOthbIzmfiEM2xS3k4o502bs4hkkn3I9F7UZGYU\nGPc+ZYBeLAgqGqx6rWVsdJTTejHkOTYRzr01n4+dYZwF1uI5QFJqqKw9LfeMQ3b1\ngLwN/FfmbBEC\n-----END CERTIFICATE-----"
        ],
        "certificate": "-----BEGIN CERTIFICATE-----\nMIICUzCCATugAwIBAgIUXOMH5Mw5jQGxFjrd2oXeKgWfkhkwDQYJKoZIhvcNAQEL\nBQAwLTErMCkGA1UEAxMibXl2YXVsdC5jb20gSW50ZXJtZWRpYXRlIEF1dGhvcml0\neTAeFw0yMjA4MjYyMDA4MThaFw0yMzAxMjMyMDA4NDhaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAATLBt8xHYkdR4kuDnNM4/vLex+vyVa8rzB3cMv7NE9N57lh\n2OHcABZbObftau9bh25VaX599fwk/EwUumGE2wnjo2MwYTAOBgNVHQ8BAf8EBAMC\nAQYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUeS8mVKwuJN2vr6ctq9rW/qhm\njcowHwYDVR0jBBgwFoAU0TbPqPx2m/VuI1KB8cLWBP64GKUwDQYJKoZIhvcNAQEL\nBQADggEBAJxJz7sDIa2SMjIsRVPfiGKf9XJQCM23MiGnz7pFVHlm5HOOkZuBpMmQ\n7oUj/1BcdBgCOAd38nzEOrpmNXBkbNTOgiqhIInPGdo4whxhSYbYi7WxJMiYFztE\n6AvbDa6Xj5pNux2fFV2Ur5iudVVCB/p3nyMHvlYqL43xyDG5BJJpDty/af9fcxUR\ny21C1z9DX2lzMMopdcPJ/W5H4jwKi/M6WtwlY5rLQ5+4HS8db5W/gcXv1uFf/OEt\nf4RtpgibBknYBsBs8jRn+7/NJw3hYwPofrWUbdk2r1MBUPYNUraxqMuaWNkDhg4W\n3Vf07lgG1Rf1v7pEItvEv0TNylu+fBs=\n-----END CERTIFICATE-----",
        "expiration": 1674504528,
        "issuing_ca": "-----BEGIN CERTIFICATE-----\nMIIDNDCCAhygAwIBAgIUG4iQ2d5V83due1fJTqnEx06D3oYwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAxMLbXl2YXVsdC5jb20wHhcNMjIwODI2MjAwNTAxWhcNMjcw\nODI1MjAwNTMxWjAtMSswKQYDVQQDEyJteXZhdWx0LmNvbSBJbnRlcm1lZGlhdGUg\nQXV0aG9yaXR5MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAq3Dw7lTU\nEZqhOPOoOZtfevxBz01PFx6BYsi3JTA7mpeSDIre0cKanwjiYc3u9z0nQF5W1Jr5\natkDUcMTOWCALAxRg5vWOq/2bDuv2lLk9f9FYIgdDQHMFQwqq4D0oGX+/ZFBVBAH\nHIYn8nMgnTnBCKzNFqHSQxCuSa42MlC1g6OO7z87sfL1wn5Inhn7OP6eoTzv5dvv\nefH6Mhvv2KvulxdA/imTaPr3wuEu1xmgg0sczUxZxIwQGU/uX+tL1Brwfow3uWGN\ngATknWly71Hp4RdeM+fFg7Or2otn6kimFV10A6kWAao1+CI7evAaPaqVXtrNGSRe\nWhZtVSAiGiZ0/QIDAQABo2MwYTAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUw\nAwEB/zAdBgNVHQ4EFgQU0TbPqPx2m/VuI1KB8cLWBP64GKUwHwYDVR0jBBgwFoAU\nUO7+pScrxAIZHhlgkoToBJHxOUEwDQYJKoZIhvcNAQELBQADggEBAEAVoshZnNsG\nebCxhzqmeAEWOV/i5SxkU4XkYumAHNlpzA6HLB1ieegRDPiMqxfiQkCCUs7WVj7f\nxnI1pZi0ZqzpR54ROJn+WqTPVp4xOxdheWjzfLzkOIfSmZtYDvXqFYS7Wghu3c/K\nno1/fIIiesAN43hzDMT9HhdLQSkCl4Lh5x8SwSZYZ1ATK/iUOwVIIBhvr9nLm+XF\ncFL89bIw9Y2g9VMi5kwrEJoiSIaq6eQkw048C/bH/sV5IhdG5zOY3J7eDxz+dY87\n/7Er+WUprmgcrp2pnwwCy9zJ3k9rzMGao9TPaCTUNl685uGqqNHZCOPscdX/v6Tz\nd5cMOisXvU8=\n-----END CERTIFICATE-----",
        "serial_number": "5c:e3:07:e4:cc:39:8d:01:b1:16:3a:dd:da:85:de:2a:05:9f:92:19"
    },
    "warnings": null
}
```

**Comment by amartinezfayo** [comment]
Hey @hiyosi, it would be great if a test case could be added to simulate the legacy behavior. Could you possibly do that?

**Comment by hiyosi** [comment]
@amartinezfayo 
Sure! added [697e72f](https://github.com/spiffe/spire/pull/3376/commits/697e72feb8afa3a9f20c7ade05a27c66330c7dc0)

**Review by amartinezfayo — APPROVED** [review]
Thank you @hiyosi!

**Pull request merged**